cmd/notary-signer/passwordwrap/interface.go

```diff
@@ -0,0 +1,19 @@
+package passwordwrap
+
+// Storage is the interface to store/retrieve password from any specific implementation type.
+type Storage interface {
+	// Get password from storage.
+	GetPassword(alias string) (string, error)
+
+	// Set password in storage.
+	SetPassword(alias string, newPassword string) error
+}
+
+// Protector is the interface to wrap/unwrap password using any specific implementation type.
+type Protector interface {
```

Comment:
In the default case, where passwords are in plaintext, one would simply not use the `Protector` at all and only use an instance of `PassStore`. This would be part of the initialization/configuration logic that happens in the main package with the instantiated objects being injected down in to the required locations.